README.md

```diff
@@ -82,6 +82,13 @@ To **disable** the responsiveness either add the `no-responsive` attribute or ch
 <a ob-lazytube no-responsive href="http://www.youtube.com/watch?v=J---aiyznGQ">Keyboard cat</a>
 ```
 
+### Custom placeholder image (instead of Youtube autogen. thumbnail)
+
+Use your own placeholder image from your server.
+
+```html
+<a ob-lazytube href="http://www.youtube.com/watch?v=J---aiyznGQ" ob-placeholder="{{'images/myVideo.png'}}">Keyboard cat</a>
```

Comment:
Yes, all three ways will still work, just that the way you wrote in the README is with superfluous characters :-) `{{}}` is only needed when you want to use an expression.